Assets/_BForBoss/_Core/Scripts/Labor/RingSystem/RingLaborManager.cs

```diff
@@ -42,9 +42,9 @@ private void CreateSystems()
             {
                 RingSystem newSystem = grouping.RingSystemType switch
                 {
-                    RingSystemTypes.Standard => new OrderedRingSystem(grouping.Rings),
-                    RingSystemTypes.DisplayAllAtOnce => new AllAtOnceRingSystem(grouping.Rings),
-                    RingSystemTypes.RandomSelection => new OrderedRingSystem(grouping.Rings, true),
+                    RingSystemTypes.Standard => new RingSystem(grouping.Rings),
+                    RingSystemTypes.DisplayAllAtOnce => new RingSystem(grouping.Rings, allAtOnce: true),
+                    RingSystemTypes.RandomSelection => new RingSystem(grouping.Rings, isRandomized: true),
```

Comment:
@Gursaanj are we still doing random? 
or are we doing predesigned only?

Reply:
We will be doing preassigned but I say lets just keep this in for now, just so we can pla

> So this change makes it so 1 ring shows up one at a time. Are we not doing all at the same time + billboarding numbers?
> 
> or is that for another ticket?
> 
> @Gursaanj

The Billboarding will be another ticket!

Reply:
We are only doing preassigned, so I think it is safe to delete the random selection

Reply:
I removed the random selection from the scene - Now the current demo loop is 5 all at once, 2 all at once, and then 5 in order sequential